ProjectLighthouse/Helpers/TimeHelper.cs

```diff
@@ -7,6 +7,8 @@ public static class TimeHelper
     public static long Timestamp => DateTimeOffset.UtcNow.ToUnixTimeSeconds();
 
     public static long TimestampMillis => DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+
+    public static long TimestampMicros => DateTimeOffset.UtcNow.ToUnixTimeMilliseconds() * 1000;
```

Comment:
This doesn't get used anywhere, if there is anything that uses microseconds just convert it to milliseconds.